.github/workflows/cypress.yml

```diff
@@ -1,12 +1,13 @@
 name: TestE2E
 
 on:
-  pull_request:
-    types: [opened, reopened, review_requested, ready_for_review, synchronize]
+  workflow_run:
+    workflows: ["Build node_modules"]
+    types: [completed]
 
 jobs:
   cypress-run: 
```

Comment:
Same, keep the "non-draft" check

Reply:
I have this condition check in the "Build node_modules" workflow. Should we double this condition ?